### PR TITLE
Refactor/improve drop table management

### PR DESCRIPTION
**Pull request by James-Mart**

### CODE REVIEW

File: src/gameLogic/useFishingTimer.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const useFishingTimer = () => {
     const isFishing = useSelector((state) => state.journey.isFishing);
 
     const startFishingTimer = () => {
-        clearTimeout(token);
```
Comment (nickhalvorsen) [inline src/gameLogic/useFishingTimer.js:17]: how did you know about this?

File: src/gameLogic/dropTable.js [hunk 2]
```diff
@@ -5,97 +5,47 @@ const dropTypes = {
     item: "item",
 };
 
-const fish = {
```
Comment (nickhalvorsen) [inline src/gameLogic/dropTable.js:8]: thanks. I was going to refactor this in the future at some point

**Comment by nickhalvorsen** [comment]
looks good, i approve it

File: src/gameLogic/useFishingTimer.js (hunk 1 above)
Comment (James-Mart) [reply src/gameLogic/useFishingTimer.js:17]: Idk, it just didn't look right, so I googled it

**Pull request merged**